### DISCUSSION TITLE
kant/user-guide/edge-unit/manage-unit

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2679] [Announcements]
# kant/user-guide/edge-unit/manage-unit

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kant/user-guide/edge-unit/manage-unit.html

**Comment by liyuerich** [comment]
还有个“等待中”

**Comment by WANG0608GitHub** [comment]
Added in PR #5357
